docx.go

```diff
@@ -3,13 +3,24 @@ package docconv
 import (
 	"archive/zip"
 	"bytes"
+	"encoding/xml"
 	"fmt"
 	"io"
 	"io/ioutil"
-	"regexp"
 	"time"
 )
 
+type TypeOverride struct {
```

Comment:
Would be better if you could hide these types in an internal package (to avoid adding internal-only exported types to the library).